No matching version found for collect.js@^4.28.6.

-   Laravel Mix Version: #.#.# (`npm list --depth=0`)
-   Node Version (`node -v`): v14.17.1
-   NPM Version (`npm -v`): 6.14.13
-   OS: Windows 10

### Description:
```
npm ERR! notarget No matching version found for collect.js@^4.28.6.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'laravel-mix'
npm ERR! notarget
```
### Steps To Reproduce:
`npm install`

> If you're seeing this error it's likely that your npm cache is broken. Please try `npm cache clean --force`.

```
$ npm cache clear --force
npm WARN using --force Recommended protections disabled.
$ npm install
npm ERR! code ETARGET
npm ERR! notarget No matching version found for collect.js@^4.28.6.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

--- a/package.json
+++ b/package.json
@@ -61,7 +61,7 @@
         "chokidar": "^3.5.1",
         "clean-css": "^4.2.3 || ^5.1.2",
         "cli-table3": "^0.6.0",
-        "collect.js": "^4.28.6",
+        "collect.js": "^4.28.5",
         "commander": "^7.2.0",
         "concat": "^1.0.3",
         "css-loader": "^5.2.6",

